.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "frontend/lib/jimmi-api-client"]
 	path = frontend/lib/jimmi-api-client
-	url = ../jimmi-api-client
+	url = https://github.com/Music-Bot-for-Jitsi/jimmi-api-client
```

Comment:
Is this change necessary?
This prevents other users from cloning the submodule using ssh as I read somewhere 😊

Reply:
fixed in c18fa2cf820b35b6abe5314af942946e0fd6cdb9